doc/parameters/param_list.md

```diff
@@ -429,15 +429,15 @@ When `controller_plugins`\`progress_checker_plugin`\`goal_checker_plugin` parame
 | ----------| --------| ------------|
```

Comment:
All the changes to this `md` file have analog updates that need to be made to navigation.ros.org under the configuration section. Each package has a page with parameter descriptions like this.